packages/schematics/angular/config/files/karma.conf.js.template

```diff
@@ -0,0 +1,39 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
```

Comment:
This seems to be a duplicate of `/build_angular/src/builders/karma/index.ts`. How are these kept in sync?

Reply:
This config file is rarely updated. 

The file however will need to be kept in sync manually since `@schematics/angular` does not have a dependency on `@angular-devkit/build-angular` and vice-versa.

Reply:
The duplication is unfortunate but needed to avoid coupling the two packages currently.
Can we add a comment to the inline config in the builder about updating this template if any changes are made?

Reply:
Added